src/mono/wasm/Wasm.Build.Tests/NonWasmTemplateBuildTests.cs

```diff
@@ -23,10 +24,10 @@ public NonWasmTemplateBuildTests(ITestOutputHelper output, SharedBuildPerTestCla
     // So, copy the reference for latest TFM, and add that back with the
     // TFM=DefaultTargetFramework
     //
-    // This is useful for the case when we are on tfm=net10.0, but sdk, and packages
-    // are really 10.0 .
-    private const string s_latestTargetFramework = "net10.0";
-    private const string s_previousTargetFramework = "net9.0";
+    // This is useful for the case when we are on latest TFM, but sdk, and packages
+    // are really the previous version .
+    private static readonly string s_latestTargetFramework = $"net{Environment.Version.Major}.0";
+    private static readonly string s_previousTargetFramework = $"net{Environment.Version.Major - 1}.0";
```

Comment:
can you please also remove the `net6.0` on line 73, 87 and 96? I don't think we need to be testing it anymore since it is EOL